README.md

```diff
@@ -17,23 +17,65 @@
     <img alt="Google Pixel running Lawnchair Launcher with green wallpaper" src="docs/assets/device-frame.png" width="250px">
 </picture>
 
-Lawnchair is a free, open-source home app for Android. Taking Launcher3 — Android’s default home app — as a starting point, it ports Pixel Launcher features and introduces rich options for customization.
+Lawnchair is a free, open-source home app for Android. Taking Launcher3—Android’s default home app—as a starting point, it ports Pixel Launcher features and introduces rich customization options.
 
-This branch houses the codebase of Lawnchair 14, which is currently in beta and is based on Launcher3 from Android 14. For Lawnchair 9 to 13, see the branches with the `9-` to `13-` prefixes, respectively.
+This branch houses the codebase of Lawnchair 15, which is currently in alpha and is based on Launcher3 from Android 15. For Lawnchair 9 to 14, see the branches with the `9-` to `14-` prefixes, respectively.
```

Comment:
Once we're in beta, it doesn't really make sense to return back to alpha. I'd suggest changing this to `beta`, or more accurately, in `development` (as we haven't released a new version yet). We can change this to `beta` once we've released a new version.